src/core/grid_based_algorithms/lb.cpp

```diff
@@ -50,6 +50,7 @@
 #include "utils/uniform.hpp"
 #include "virtual_sites/lb_inertialess_tracers.hpp"
 
+#define R123_USE_MULHILO64_C99 1
```

Comment:
this should go to random.hpp

Reply:
Why random.hpp? It needs to be defined before including the Philox header. For lb_particle_coupling.cpp, it actually has to go right where it is because the Philox header is included before anything else.

Reply:
If this should be used everywhere with philox you could also put a define on the cmake target, instead of putting it into a header.